src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -4061,7 +4061,9 @@ namespace Akka.Routing
     }
     public abstract class Group : Akka.Routing.RouterConfig, System.IEquatable<Akka.Routing.Group>
     {
+        protected readonly string[] _paths;
```

Comment:
Is it really intended? ;)

Reply:
Yep - the value has to be stored somewhere in order to be propagated inside local routers. We kind of effed ourselves by using an automatic property to set the value before.

Reply:
If it's intended to work this way, please correct the naming conventions.

Reply:
@Horusiath done - renamed it to fix the ReSharper conventions.